scheduler/mesosphere-dcos/upgrade-1.1.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "Upgrade Portworx on DC/OS"
```

Comment:
The travis build is failing because this page has the same title as the old page. Add (Deprecated) or (Old Framework) or something similar